src/main/java/org/apache/datasketches/theta/HeapAnotB.java

```diff
@@ -99,10 +99,9 @@ public CompactSketch getResult(final boolean dstOrdered, final WritableMemory ds
       Arrays.sort(compactCache);
     }
     //Create the CompactSketch
-    final boolean empty = (curCount_ == 0) && (thetaLong_ == Long.MAX_VALUE);
     final CompactSketch comp = createCompactSketch(
-        compactCache, empty, seedHash_, curCount_, thetaLong_, dstOrdered, dstMem);
-    reset();
+        compactCache, empty_, seedHash_, curCount_, thetaLong_, dstOrdered, dstMem);
+    reset(); //TODO
```

Comment:
not clear what to do here

Reply:
That is a placeholder for the next round of changes.  Just ignore.
Thanks for the review!